src/main/java/com/thecookiezen/bussiness/cluster/entity/DockerHost.java

```diff
@@ -6,19 +6,23 @@
 
 import java.io.Serializable;
 import java.time.Instant;
+import java.util.Random;
 
 @Data
-@NoArgsConstructor
 public class DockerHost implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    private long id;
+    private final long id;
 
     @NonNull
     private String name;
 
     @NonNull
     private String dockerDaemonUrl;
 
-    private Instant created = Instant.now();
+    private final Instant created = Instant.now();
+
+    public DockerHost() {
+        id = new Random().nextLong();
```

Comment:
\[Checkstyle\] WARNING: 'ctor def' child have incorrect indentation level 8, expected level should be 4\.